Pipfile

```diff
@@ -26,7 +26,7 @@ django-debug-toolbar = "*"
 django-cors-headers = "*"
 flake8 = "*"
 django_compressor = "==2.1.1"
-Django = "==1.11.20"
+Django = "==1.11.21"
```

Comment:
Can probs tighten up our dependencies and remove this from dev dependencies

Reply:
Maybe cleaning up Pipfile should have its own ticket